charts/fe-nextjs/templates/_helpers.tpl

```diff
@@ -0,0 +1,77 @@
+{{/*
```

Comment:
I imagined in this file also a section defining common labels, as per the standard [casavo labelling rules](https://casavo.atlassian.net/wiki/spaces/cd/pages/1435467786/Kubernetes+Labeling)

Reply:
How should I know those labels in advance? Will be added in values by projects IMHO

Reply:
@edubacco also: remember all of this is **public** ;)

Reply:
Let's talk about abstraction layer for required labels in CoP Devops